modules/mobile/app/controllers/mobile/v0/maintenance_windows_controller.rb

```diff
@@ -24,7 +24,8 @@ class MaintenanceWindowsController < ApplicationController
         %i[mhv secure_messaging],
         %i[vaos appointments],
         %i[vet360 user_profile_update],
-        %i[mhv rx_refill]
+        %i[mhv rx_refill],
+        %i[eoas preneed_burial]
```

Comment:
I looked through the logs and see no instances of an eoas putting windows in vets-api. I figured it'd be better to include it anyway for if it does get added later.